src/borda_clj/client.clj

```diff
@@ -41,12 +50,12 @@
    Parameter global-dimensions is a map of dimensions that will be included with
    every single measurement."
   [global-dimensions max-buffer-size interval send on-send-error]
-  (let [measurements  (atom (empty-measurements))
+  (let [measurements  (atom {})
         running       (atom true)
-        submit        (fn [dimensions values] (swap! measurements collect max-buffer-size dimensions values))
-        resubmit      (fn [m] (doseq [[dimensions values] m] (submit dimensions values)))
+        submit        (fn [count_measure dimensions values] (swap! measurements collect count_measure max-buffer-size dimensions values))
+        resubmit      (fn [m] (doseq [[dimensions values] m] (submit false dimensions values)))
```

Comment:
Why?  Measurements dropped on resubmit are still dropped.

Reply:
Yeah, this logic was broken. New logic is better I think.